src/components/Footer/Fotter.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  status: string;
+  onStatusChange: (arg: string) => void;
+  clearCompletedTodos: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  status,
+  onStatusChange,
+  clearCompletedTodos,
+}) => {
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const isAnyCompleted = todos.some(todo => todo.completed);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodos.length} items left`}
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: status === 'all',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => onStatusChange('all')}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: status === 'active',
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => onStatusChange('active')}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: status === 'completed',
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => onStatusChange('completed')}
+        >
+          Completed
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method